<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
    background-color: rgb(0, 0, 0);
    font-family: Arial;
    color: rgb(255, 255, 255);
}

QMenuBar{
    text-decoration: underline rgb(255, 255, 255);
}

QLabel{
    font: 75 18pt;
}

QLineEdit,QComboBox,QDoubleSpinBox,QSpinBox{
    color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
    font: 75 14pt &quot;Arial&quot;;
}

QPushButton{
  color: #fff !important;
  font-size: 20px;
  font-weight: 500;
  background: #318aac;
  border: 2px solid;
  border-color: #318aac;
  position: relative;
}

QPushButton:before {
  content:&quot;&quot;;
  position: absolute;
  top: 0px;
  left: 0px;
  width: 100%;
  height: 0px;
  background: rgba(255, 255, 255, 0.1);
  transition: all 1s ease;
}

QPushButton:hover:before {
  height: 100%;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Eliminar cita</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ID del paciente</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de cita</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_idp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_eliminar">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar cita</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla_citas">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>611</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID cita</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID paciente</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido Paterno</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido Materno</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha de cita</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="boton_buscar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar cita</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ID de cita</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_idc">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_fecha">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionRegresar"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRegresar">
   <property name="text">
    <string>Regresar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
